portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,10 +76,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
       // Get the input from the form
-      String comment = request.getParameter("comment");
-      Entity commentEntity = new Entity("Comment");
-      commentEntity.setProperty("comment", comment);
-      datastore.put(commentEntity);
+      String comment = request.getParameter(PROPERTY_COMMENT);
```

Comment:
looks like some of these changes are addressing feedback from the previous PR; going forward, update the original PR branch with the changes before submitting in order to keep the diffs tidy :)